src/Impostor.Api/Innersloth/Data/maps/Airship/spawn.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this omits the airshit spawn picking minigame

Reply:
Which is how base game does it aswell, all shipstatus prefabs have to have all of those fields set to *something*, and then it can be overriden in `ShipStatus#GetSpawnLocation`